apps/api/src/domain/common/events/coscrad-event-consumer.decorator.spec.ts

```diff
@@ -0,0 +1,133 @@
+import { AggregateType } from '@coscrad/api-interfaces';
+import { Injectable } from '@nestjs/common';
+import { ConsoleCoscradCliLogger } from '../../../coscrad-cli/logging';
+import { BaseEvent } from '../../../queries/event-sourcing';
+import buildDummyUuid from '../../models/__tests__/utilities/buildDummyUuid';
+import { dummyDateNow } from '../../models/__tests__/utilities/dummyDateNow';
+import { dummySystemUserId } from '../../models/__tests__/utilities/dummySystemUserId';
+import { CoscradEventConsumer } from './coscrad-event-consumer.decorator';
+import { ICoscradEventHandler } from './coscrad-event-handler.interface';
+import { ICoscradEvent } from './coscrad-event.interface';
+import { SyncInMemoryEventPublisher } from './sync-in-memory-event-publisher';
+
+export interface HasHandles {
+    handles(e: BaseEvent): boolean;
+}
+
+describe('@CoscradEventConsumer', () => {
```

Comment:
I think this is the way we want to go when implementing general infrastructure. We create toy models that leverage the tooling (decorator in this case) instead of coupling to our real models, which are certain to change independent of the behaviour this test covers.